src/vessel-charging/messages/ChargingCompleteMessageParams.test.ts

```diff
@@ -1,16 +1,19 @@
 import MessageParams from './ChargingCompleteMessageParams';
 
 describe('MessageParams class', () => {
-  const messageParams = new MessageParams({
-    senderId: 'TOPIC_ID',
-  });
+  let messageParams: MessageParams;
+  let serializedMessageParams: any;
+
+  beforeEach(() => {
+    messageParams = new MessageParams({});
 
-  const serializedMessageParams: any = {
-    ttl: undefined,
-    protocol: 'vessel_charging',
-    type: 'charging_complete_message',
-    senderId: 'TOPIC_ID',
-  };
+    serializedMessageParams = {
+      ttl: undefined,
+      protocol: 'vessel_charging',
+      type: 'charging_complete_message',
+      senderId: 'TOPIC_ID',
+    }
```

Comment:
There's a semicolon missing at the end of this line.

If you run `npm run tslint` in the project directory you should see the error message.

<img width="983" alt="job__211_1_-_davfoundation_dav-js_-_travis_ci" src="https://user-images.githubusercontent.com/926653/46278762-cd3a6c80-c56f-11e8-9220-25052f1cf133.png">

This is the reason the build failed